events/scheme_events/stasis_trap_scheme/wc_stasis_trap_outcome_events.txt

```diff
@@ -0,0 +1,351 @@
+﻿namespace = stasis_trap_outcome
+
+##################################
+# MAINTENANCE EVENTS 0000 - 0099
+##################################
+
+stasis_trap_outcome.0001 = {
+	hidden = yes
+	immediate = {
+		add_character_flag = { #Only resolve one stasis trap in the same day
+			flag = is_in_event_stasis_trap_outcome_0001
+			days = 1
+		}
+		trigger_event = {
+			on_action = stasis_trap_setup
+		}
+	}
+}
+
+#################
+# METHOD SELECTION EVENTS
+# 1000-1999
+#################
+
+######################
+# Trapped Residence 
+######################
+stasis_trap_outcome.1001 = {
+	type = character_event
+	title = stasis_trap_outcome.1001.t
+	desc = {
+		desc = stasis_trap_outcome.1001.desc
+	}
+	theme = intrigue
+	left_portrait = {
+		character = scope:target
+		animation = personality_coward
+	}
+	widget = {
+		gui = "event_window_widget_scheme"
+		container = "custom_widgets_container"
+	}
+
+	weight_multiplier = {
+		base = 1
+	}
+
+	immediate = {
+		save_scope_value_as = {
+			name = stasis_trap_method
+			value = flag:trapped_residence
+		}
+		
+		scope:target.court_owner.capital_province.barony = {
+			save_scope_as = court_location
+		}
+
+		hostile_scheme_outcome_roll_setup_effect = yes
+	}
+
+	option = {
+		name = stasis_trap_outcome.1001.a
+		remove_short_term_gold = minor_gold_value
+		stasis_trap_outcome_roll_effect = yes
+		ai_chance = {
+			base = 100
+		}
+	}
+
+	option = {
+		name = stasis_trap_outcome.1001.b
+		custom_tooltip = do_not_execute_stasis_trap_tooltip
+
+		dont_excecute_scheme_effect = yes
+
+		ai_chance = {
+			base = 0
+		}
+	}
+}
+
+#####################################
+# SUCCESS OUTCOMES 2001 - 2999
+#####################################
+
+#######################
+# Trapped Residence
+#######################
+stasis_trap_outcome.2001 = {
+	type = character_event
+	title = stasis_trap_outcome.1001.t
+	desc = {
+		desc = stasis_trap_outcome.2001.desc
+
+		first_valid = {
+			# If the Scheme Owner is known:
+			triggered_desc = {
+				trigger = { 
+					OR = {
+						AND = {
+							exists = scope:scheme_discovered
+							scope:scheme_discovered = yes 
+						}
+						scope:scheme = {
+							is_scheme_exposed = yes
+						}
```

Comment:
Exposed doesn't mean the owner is known. It means the target knows this scheme exists without implying the owner is known.